src/components/navigation-disclosure/components/navigation-disclosure-content/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { DisclosureContent } from 'components/disclosure'
+import { validateNavigationDisclosureContentChildren } from 'components/navigation-disclosure/helpers'
+import { NavigationDisclosureContentProps } from './types'
+
+/**
+ * Component for rendering content within a `NavigationDisclosure`.
+ *
+ * @see https://developer.hashi-mktg.com/swingset/components/navigationdisclosure
+ */
+const NavigationDisclosureContent = ({
+  children,
+  className,
+}: NavigationDisclosureContentProps) => {
+  validateNavigationDisclosureContentChildren(children)
+
+  return <DisclosureContent className={className}>{children}</DisclosureContent>
+}
+
```

Comment:
question: Will there be more to some of these pass-through components? It seems like we could use the underlying disclosure components, but this is understandable if they are going to diverge.

Reply:
It's very possible they will! This component's main responsibility right now is ensuring that the correct component is passed to it (`NavigationDisclosureList `).

Another responsibility it has is that it's essentially a `DisclosureContent` namespaced to `NavigationDisclosureContent`. I think this helps two cases in particular:

- Someone onboarding to the codebase that is tasked with using `NavigationDisclosure` but doesn't know about `Disclosure` yet. As is, all the components needed to render a `NavigationDisclosure` all live next to each other. No need to know that `Disclosure` even exists to get started.
- I'd find the code much harder to follow having a mix of component names rather than all the component names having the same prefix/namespace. Here's an example (a little verbose since a `map` isn't being used but hopefully still communicates my perspective):

  ```tsx
  const MyComponent = () => {
    return (
      <NavigationDisclosure>
        <DisclosureActivator>Activator Text</DisclosureActivator>
        <DisclosureContent>
          <NavigationDisclosureList>
            <NavigationDisclosureListItem>
              <NavigationDisclosureLink>First link</NavigationDisclosureLink>
            </NavigationDisclosureListItem>
            <NavigationDisclosureListItem>
              <NavigationDisclosureLink>Second link</NavigationDisclosureLink>
            </NavigationDisclosureListItem>
            <NavigationDisclosureListItem>
              <NavigationDisclosureLink>Third link</NavigationDisclosureLink>
            </NavigationDisclosureListItem>
          </NavigationDisclosureList>
        </DisclosureContent>
      </NavigationDisclosure>
    )
  }
  ```